packages/ninja.rb

```diff
@@ -3,30 +3,40 @@
 class Ninja < Package
   description 'a small build system with a focus on speed'
   homepage 'https://ninja-build.org'
-  version '1.10.0'
+  version '1.10.2'
   license 'GPL-2'
   compatibility 'all'
-  source_url 'https://github.com/ninja-build/ninja/archive/v1.10.0.tar.gz'
-  source_sha256 '3810318b08489435f8efc19c05525e80a993af5a55baa0dfeae0465a9d45f99f'
+  source_url 'https://github.com/ninja-build/ninja/archive/v1.10.2.tar.gz'
+  source_sha256 'ce35865411f0490368a8fc383f29071de6690cbadc27704734978221f25e2bed'
 
-  binary_url ({
-    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/ninja/1.10.0_armv7l/ninja-1.10.0-chromeos-armv7l.tar.xz',
-     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/ninja/1.10.0_armv7l/ninja-1.10.0-chromeos-armv7l.tar.xz',
-       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/ninja/1.10.0_i686/ninja-1.10.0-chromeos-i686.tar.xz',
-     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/ninja/1.10.0_x86_64/ninja-1.10.0-chromeos-x86_64.tar.xz',
+  binary_url({
+    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/ninja/1.10.2_armv7l/ninja-1.10.2-chromeos-armv7l.tar.xz',
+     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/ninja/1.10.2_armv7l/ninja-1.10.2-chromeos-armv7l.tar.xz',
+       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/ninja/1.10.2_i686/ninja-1.10.2-chromeos-i686.tar.xz',
+     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/ninja/1.10.2_x86_64/ninja-1.10.2-chromeos-x86_64.tar.xz'
   })
-  binary_sha256 ({
-    aarch64: '5e2ef70ad213e86a4a32e48bb4783face0bff6f3b079080d3d668bb35ba83088',
-     armv7l: '5e2ef70ad213e86a4a32e48bb4783face0bff6f3b079080d3d668bb35ba83088',
-       i686: 'ec2996217bdf7c57d0528a8ba0eefabda15c17e59f958daebc3fe1eda6a8a9bc',
-     x86_64: 'e74f2b1d374edd0ff0877939b3afb8fa80240774bf3c9126278e5a965c7a5d0b',
+  binary_sha256({
+    aarch64: 'd6fe1cfab1c8c0c9a4865fcc1918ebe86a4effd17a28d9fc7b90401e550e2b90',
+     armv7l: 'd6fe1cfab1c8c0c9a4865fcc1918ebe86a4effd17a28d9fc7b90401e550e2b90',
+       i686: '36f809229fd7779f0783742a579043c9b8a69ec9454efcb1ff7655fd621d5e45',
+     x86_64: '40ee7476d70e15874acb59b43e4e379c67cc65793ead2af6fbd67331f06193ff'
   })
 
+  depends_on 're2c' => :build
+
+  def self.patch
+    system 'filefix'
+  end
+
   def self.build
-    system 'python3 configure.py --bootstrap'
+    system "env #{CREW_ENV_OPTIONS} \
+      AR=gcc-ar \
+      python3 configure.py --bootstrap"
+    system 'ninja'
```

Comment:
Do you mean `system './ninja'`?

Reply:
Let me check...

Reply:
Turns out we don't need that line at all!